src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -74,6 +74,9 @@ public Command parseCommand(String userInput) throws ParseException {
         case ExitCommand.COMMAND_WORD:
             return new ExitCommand();
 
+        case ContactCommandParser.COMMAND_WORD:
+            return new ContactCommandParser().parse(arguments);
+
```

Comment:
If we are to delete `DeleteCommand.java` file, I think it will be optimal to also remove line 62-63
`case DeleteCommand.COMMAND_WORD:
return new DeleteCommandParser().parse(arguments); `
as we do not want it to be executed and do unintended actions

Reply:
It may be viable. I will do such deletion in the list command pull request, but it also means that many tests become irrelevant...